torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1627,8 +1652,11 @@ void QuantFusion(std::shared_ptr<Graph>& graph) {
 
 script::Module FoldConvBatchNorm2d(const script::Module& module) {
   FoldConvBatchNorm2dHelper h;
-  h.run(module);
-  return module;
+  script::Module m = module.clone();
+  h.analyze(m);
+  h.transform();
+  h.cleanup();
```

Comment:
While I like the split into analysis and transformation, we need to add comments what exactly is happening when we're doing this. Please describe what data the analysis collects/verifies, what properties transformation expects, what we're doing in the cleanup. Without this, the code is not much better than
```
doStuff();
doMoreStuff();
doExtraStuff();
```

Reply:
Sure, will add more comments